modules/api-resources/api-resources-full/src/main/webapp/WEB-INF/beans.xml

```diff
@@ -125,23 +125,24 @@
     </jaxrs:server>
     <jaxrs:server id="users" address="/users/v1">
         <jaxrs:serviceBeans>
+            <bean class="org.wso2.carbon.identity.rest.api.user.application.v1.MeApi"/>
+            <bean class="org.wso2.carbon.identity.rest.api.user.approval.v1.MeApi"/>
             <bean class="org.wso2.carbon.identity.rest.api.user.association.v1.MeApi"/>
+            <bean class="org.wso2.carbon.identity.rest.api.user.association.v1.UserIdApi"/>
             <bean class="org.wso2.carbon.identity.rest.api.user.authorized.apps.v1.MeApi"/>
+            <bean class="org.wso2.carbon.identity.rest.api.user.authorized.apps.v1.UserIdApi"/>
+            <bean class="org.wso2.carbon.identity.rest.api.user.backupcode.v1.MeApi"/>
             <bean class="org.wso2.carbon.identity.rest.api.user.challenge.v1.MeApi"/>
-            <bean class="org.wso2.carbon.identity.rest.api.user.approval.v1.MeApi"/>
-            <bean class="org.wso2.carbon.identity.rest.api.user.session.v1.MeApi"/>
+            <bean class="org.wso2.carbon.identity.rest.api.user.challenge.v1.UserIdApi"/>
             <bean class="org.wso2.carbon.identity.rest.api.user.fido2.v1.MeApi"/>
-            <bean class="org.wso2.carbon.identity.rest.api.user.totp.v1.MeApi"/>
-            <bean class="org.wso2.carbon.identity.rest.api.user.backupcode.v1.MeApi"/>
+            <bean class="org.wso2.carbon.identity.rest.api.user.functionality.v1.MeApi"/>
+            <bean class="org.wso2.carbon.identity.rest.api.user.functionality.v1.UserIdApi"/>
```

Comment:
why introduced these now? Didn't we have these ?

Reply:
This is a alphabetic re-ordering